ipatests/test_webui/test_pwpolicy.py

```diff
@@ -54,4 +51,214 @@ def test_crud(self):
         Basic CRUD: pwpolicy
         """
         self.init_app()
-        self.basic_crud(ENTITY, DATA)
+        self.basic_crud(pwpolicy.ENTITY, pwpolicy.DATA)
+
+    @screenshot
+    def test_misc(self):
+        """
+        various test cases covered in one place
+        """
+        # Basic requirement : create user group for test
+        self.init_app()
+        self.navigate_to_entity(group.ENTITY)
+        self.add_record(group.ENTITY, [group.DATA, group.DATA2, group.DATA3,
+                                       group.DATA4, group.DATA5, group.DATA6,
+                                       group.DATA_SPECIAL_CHAR_GROUP])
+
+        # add then cancel
+        self.add_record(pwpolicy.ENTITY, pwpolicy.DATA1, dialog_btn='cancel')
+
+        # test add and add another record
+        self.add_record(pwpolicy.ENTITY, [pwpolicy.DATA1, pwpolicy.DATA2])
+
+        # test add and edit record
+        self.add_record(pwpolicy.ENTITY, pwpolicy.DATA,
+                        dialog_btn='add_and_edit')
+
+        # test add password policy for special characters group
+        self.add_record(pwpolicy.ENTITY, pwpolicy.DATA_SPECIAL_CHAR,
+                        delete=True)
+
+        # empty group and priority (requires the field)
+        self.navigate_to_entity(pwpolicy.ENTITY)
+        self.facet_button_click('add')
+        self.dialog_button_click('add')
+        self.assert_field_validation_required(field='cn')
+        self.assert_field_validation_required(field='cospriority')
+        self.close_all_dialogs()
+
+        # test delete default policy and
+        # confirming by keyboard to test ticket-4097
+        self.navigate_to_entity(pwpolicy.ENTITY)
+        self.select_record(pwpolicy.DEFAULT_POLICY)
+        self.facet_button_click('remove')
+        self.dialog_button_click('ok')
+        self.assert_last_error_dialog(EXPECTED_ERR, details=True)
+        actions = ActionChains(self.driver)
+        actions.send_keys(Keys.TAB)
+        actions.send_keys(Keys.ENTER).perform()
+        self.wait(0.5)
+        self.assert_record(pwpolicy.DEFAULT_POLICY)
+
+        # test add/delete Passwordpolicy confirming using ENTER key ticket-3200
```

Comment:
a nit pick, wrt `ticket-3200`, we are not very consistent in source code when referencing tickets but this looks like yet another notation.  Could we stick to something already used? E.g. `ticket 3200` or `ticket #3200`.